src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.enums.LottoWinnerPrice;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoResult {
+
+    private final List<Integer> result;
+    private final double returnRate;
+
+    public double getReturnRate() {
+        return returnRate;
+    }
+
+    public LottoResult(List<Integer> result, int buyPrice) {
+        this.result = result;
+        this.returnRate = Math.round((double) getPriceTotal() / buyPrice * 100.0) / 100.0;
+    }
+
+    int getPriceTotal() {
+        int result = 0;
+        for (Integer matchedNumber : this.result) {
+            result += LottoWinnerPrice.getPrice(matchedNumber);
+        }
+        return result;
+    }
+
+    public int getWinnerCount(int matchedNumber) {
+        return (int) result.stream()
+                .filter(count -> count == matchedNumber)
+                .count();
+    }
+
+    public static LottoResult getLottoResult(List<Lotto> lottos, Lotto winner, int lottoPrice) {
```

Comment:
```suggestion
    public static LottoResult getLottoResult(List<Lotto> lottos, Lotto winner) {
```
로또 price를 인자로 받을 필요가 있을까요?
고정값이니 상수로 바로 사용해도 될 것 같아요

Reply:
수정했습니다!